docs/pages/index.vue

```diff
@@ -305,34 +252,9 @@
 
           <b-card tag="article" body-text-variant="muted" body-class="px-2" class="rounded-0 border-0">
             <b-card-text text-tag="h3" class="h5 text-center bd-text-purple-bright mb-2">
-              <svg
-                xmlns="http://www.w3.org/2000/svg"
-                height="2.5em"
-                viewBox="0 0 24 24"
-                fill="none"
-                stroke="currentColor"
-                stroke-width="2"
-                stroke-linecap="round"
-                stroke-linejoin="round"
-                class="mx-auto d-block mb-3"
-                aria-hidden="true"
-                focusable="false"
-                role="img"
-              >
-                <line x1="4" y1="21" x2="4" y2="14" />
-                <line x1="4" y1="10" x2="4" y2="3" />
-                <line x1="12" y1="21" x2="12" y2="12" />
-                <line x1="12" y1="8" x2="12" y2="3" />
-                <line x1="20" y1="21" x2="20" y2="16" />
-                <line x1="20" y1="12" x2="20" y2="3" />
-                <line x1="1" y1="14" x2="7" y2="14" />
-                <line x1="9" y1="8" x2="15" y2="8" />
-                <line x1="17" y1="16" x2="23" y2="16" />
-              </svg>
-
+              <b-icon icon="tools" width="2.5em" height="2.5em" class="mx-auto d-block mb-3"></b-icon>
```

Comment:
I would suggest to use the `gear` or `gear-fill` icon.

Reply:
@tmorehouse Other than that I'am fine with the changes.

Reply:
OK.. trying the `gear-fill` icons

Reply:
Maybe `gear` fits better with the other outline icons.

Reply:
Yeah

Reply:
Much better.